### PR TITLE
dev-vcs/tig: Add version bump 2.3.0

### PR DESCRIPTION
**Pull request by GuillaumeSeren**
Closes: https://bugs.gentoo.org/632530
Gentoo-Bug: 632530

### CODE REVIEW

File: dev-vcs/tig/tig-2.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/jonas/tig.git"
+	inherit git-r3 autotools
+else
+	SRC_URI="https://github.com/jonas/tig/releases/download/${P}/${P}.tar.gz"
+	KEYWORDS="~alpha amd64 arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+fi
+
+DESCRIPTION="text mode interface for git"
```
Comment (jonasstein) [inline dev-vcs/tig/tig-2.3.0.ebuild]: A text mode interface for git.

Comment (GuillaumeSeren) [reply dev-vcs/tig/tig-2.3.0.ebuild]: @jonasstein 
I can fix it but it comes from the title of the project on GitHub: "Text-mode interface for git" from https://github.com/jonas/tig

File: dev-vcs/tig/tig-2.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+EAPI=6
+
+inherit bash-completion-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/jonas/tig.git"
+	inherit git-r3 autotools
+else
+	SRC_URI="https://github.com/jonas/tig/releases/download/${P}/${P}.tar.gz"
+	KEYWORDS="~alpha amd64 arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
```
[the remark this answers is not in the document]
Comment (GuillaumeSeren) [reply dev-vcs/tig/tig-2.3.0.ebuild]: Ah didn't know this tool, it is nice thank you for the pointer.

File: dev-vcs/tig/tig-2.3.0.ebuild (hunk 1 above)
Comment (GuillaumeSeren) [reply dev-vcs/tig/tig-2.3.0.ebuild]: Ok I upgraded it !

**Comment by GuillaumeSeren** [comment]
@jonasstein 
I upgraded the branch !

**Comment by GuillaumeSeren** [comment]
Hey,
I am coming for news is there anything I should change to to get this bump moving ?

**Comment by candrews** [comment]
d78cb57faf50ccb763addb20ebd667a784dfe56d

**Pull request closed without merging**